code/game/gamemodes/events/powercreeper.dm

```diff
@@ -45,23 +46,19 @@
 	//basic cable stuff, this gets done in the cable stack logic, so i needed to copy paste it over, oh well
 	var/datum/powernet/PN = getFromPool(/datum/powernet)
 	PN.add_cable(src)
+	//we are processing
+	processing_objects.Add(src)
+	. = ..()
+	set_light(1, 15, LIGHT_COLOR_RED)
```

Comment:
why are you no longer spawning set_light?

Reply:
Because the reason it wasn't being placed was because the thing hadn't been "placed" yet. Hence it is now after the supercall.

Reply:
i see, it will still hold up the powernet connecting tho, altough that isn't as big of an issue